examples/render/contour-gateway.yaml

```diff
@@ -3167,3 +3167,3340 @@ spec:
           secret:
             secretName: envoycert
       restartPolicy: Always
+
```

Comment:
I think we need a `---` separator here; the component Contour files all include them but Gateway CRDs don't have one at the top